src/pages/calendar/components/AddTask/RadioButton.jsx

```diff
@@ -0,0 +1,74 @@
+import { useTranslation } from 'react-i18next'
+import styled, { css } from 'styled-components'
+
+export const RadioButton = ({ title, checked, onChange }) => {
+  const { t } = useTranslation();
+  let radioColor = '';
+  let priority=title.charAt(0).toUpperCase() + title.slice(1);
+
+  switch (title) {
+    case 'low':
+      radioColor = '114, 194, 248'
+      break
+    case 'medium':
+      radioColor = '243,178,73'
+      break
+    case 'high':
+      radioColor = '234,61,101'
+      break
+    default: radioColor='114, 194, 248'
+      break
+  }
+  
+  return (
+    <RadioLabel htmlFor={title}>
+      <RadioInput
+        type='radio'
+        value={title}
+        id={title}
+        checked={checked === title}
+        onChange={onChange}
+        radioColor={radioColor}
+      />
+      <span>{t(priority)}</span>
+    </RadioLabel>
+  )
+}
+
+
+const RadioInput = styled.input`
+ ${({ theme: { colors } }) => css`
+  position: absolute;
```

Comment:
Why do you need position absolute?

Reply:
The size of the radiobuttons differs in checked and unchecked state.